docs/en/operation/Troubleshooting.md

```diff
@@ -51,7 +51,7 @@ will need to set the JVM remote debugging parameters before starting the process
 the remote debugging parameters; you can export the following configuration properties in shell or `conf/alluxio-env.sh`:
 
 ```shell
-# Java 5 through 8
```

Comment:
I believe the lowest version we support is 8

Reply:
yes pls